discovery-provider/src/models/models.py

```diff
@@ -568,6 +568,50 @@ def __repr__(self):
 created_at={self.created_at})>"
 
 
+class PlaysArchive(Base):
+    __tablename__ = "plays_archive"
+
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, nullable=True, index=False)
+    source = Column(String, nullable=True, index=False)
+    play_item_id = Column(Integer, nullable=False, index=False)
+    slot = Column(Integer, nullable=True, index=True)
+    signature = Column(String, nullable=True, index=False)
+    created_at = Column(DateTime, nullable=False, default=func.now())
+    updated_at = Column(
+        DateTime, nullable=False, default=func.now(), onupdate=func.now()
+    )
+    archived_at = Column(
+        DateTime, nullable=False, default=func.now(), onupdate=func.now()
+    )
+
+    Index("plays_archive_created_at_idx", "created_at")
```

Comment:
Do we want all these indexes? Won't that slow down insertions a lot and we don't plan to query it? Or will you think about dropping these at a later time?

Reply:
Yeah I was debating this. It was pretty fast but my prod snapshot wasn't under load. I'll drop `INCLUDING ALL` in the migration which will remove the indexes.